lib/rules/index.js

```diff
@@ -93,4 +93,5 @@ module.exports = {
   'style-concatenation': require('./style-concatenation'),
   'table-groups': require('./table-groups'),
   'template-length': require('./template-length'),
+  'require-has-block-helper': require('./require-has-block-helper'),
```

Comment:
I think you need to alphabetize this one (the generator always adds it to the bottom, but then we fail a test if they aren't alphabetized).